tests/integration-tests/clusters_factory.py

```diff
@@ -54,13 +55,22 @@ def update(self, config_file, force=True):
         """
         # update the cluster
         logging.info("Updating cluster {0} with config {1}".format(self.name, config_file))
-        command = ["pcluster", "update", "--config", config_file]
+        command = [
+            "pcluster",
+            "update-cluster",
+            "--cluster-configuration",
+            config_file,
+            "--cluster-name",
+            self.name,
+            "--wait",
+        ]
         if force:
-            command.append("--force")
-        command.append(self.name)
+            command.extend(["--force-update", "true"])
         result = run_command(command)
-        if "Status: {0} - UPDATE_COMPLETE".format(self.cfn_name) not in result.stdout:
-            error = "Cluster update failed for {0} with output: {1}".format(self.name, result.stdout)
+        logging.info("update-cluster --wait response: %s", result.stdout)
+        response = json.loads(result.stdout)
+        if response.get("cloudFormationStatus") != "UPDATE_COMPLETE":
+            error = f"Cluster update failed for {self.name}"
```

Comment:
Should we include the json output in the error?

Otherwise I would combine this line with the following

Reply:
The output is already printed on Line 70: ` logging.info("update-cluster --wait response: %s", result.stdout)` I wanted to print the output before `json.loads` in case `json.loads` fails.

The `error` string is used both on Line 74 and 75, so it would ok to keep the it. That said, I am not sure about the necessity of putting the `error` in Exception on Line 75.

Reply:
Ah okay --  yeah that all checks out. Seems reasonable to keep it as-is. :+1: